src/Slider.jsx

```diff
@@ -572,33 +581,48 @@ export default React.createClass({
   },
 
   render() {
+    const {
+      algorithm,
+      children,
+      disabled,
+      handle: Handle,
+      handleTabIndexStart,
+      max,
+      min,
+      orientation,
+      pitComponent: PitComponent,
```

Comment:
We should probably rename our component props to be capitalized sometime soon :-)